examples/admipex9_cb.jl

```diff
@@ -0,0 +1,21 @@
+using CPLEX
+
+env = CPLEX.Env()
+# This is pretty important for thread safety (probably).
+CPLEX.set_param!(env, "CPX_PARAM_THREADS", 1)
```

Comment:
@chengg04 make sure you set this for now. Once things are wrapped we can have a think about what happens if we let CPLEX use multiple threads...